NitroxTest/Client/Communication/MultiplayerSessionTests/TestConstants.cs

```diff
@@ -1,14 +1,15 @@
 ﻿using NitroxClient.Communication.Abstract;
 using NitroxModel.MultiplayerSession;
 using NitroxModel.Packets;
+using NitroxModel.Serialization;
 using NSubstitute;
 
 namespace NitroxTest.Client.Communication.MultiplayerSessionTests
 {
     internal static class TestConstants
     {
         public const string TEST_IP_ADDRESS = "#.#.#.#";
-        public const int TEST_SERVER_PORT = 11000;
+        public const int TEST_SERVER_PORT = ServerList.DEFAULT_PORT;
```

Comment:
Minor: Would revert this because it's a testing constant and shouldn't depend on the assembly to test.